diff --git a/node_modules/react-quill/lib/component.js b/node_modules/react-quill/lib/component.js
index 773f550..6959b8a 100644
--- a/node_modules/react-quill/lib/component.js
+++ b/node_modules/react-quill/lib/component.js
@@ -5,7 +5,7 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
 var Quill = require('quill');
-var isEqual = require('lodash/isEqual');
+var isEqual = require('lodash.isequal');
 var T = require('prop-types');
 
 var QuillComponent = React.createClass({
